### PR TITLE
[TwigBridge] Fix deprecation on twig 2.9

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes$
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

should make travis green

### CODE REVIEW

**Comment by ro0NL** [comment]
seeing https://github.com/twigphp/Twig/blob/2e68b5fdc4b57860241528b4961cbbb5d3c7a917/src/NodeVisitor/NodeVisitorInterface.php#L34 was a bit confusing, it includes the deprecated type (false) still right?

File: src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php [hunk 1]
```diff
@@ -95,7 +96,8 @@ protected function doEnterNode(Node $node, Environment $env)
     protected function doLeaveNode(Node $node, Environment $env)
     {
         if ($node instanceof TransDefaultDomainNode) {
-            return false;
+            // must return null as of Twig 2.9
```
Comment (ro0NL) [inline src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php]:
```suggestion
            // must return null as of Twig 1.40 / 2.9
```

**Review by xabbuh — APPROVED** [review]
~~with the tiny suggestion made by @ro0NL~~ (looks like it doesn't matter for 1.40 as the deprecation is not part of the `1.x` branch)

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @chalasr.

**Pull request merged**